test/matchers/mock/toBeCalledTimes.test.ts

```diff
@@ -79,21 +80,21 @@ describe('toBeRequestedTimes', () => {
 
         // expect(mock).not.toBeRequestedTimes(0) should fail
         const result = await toBeRequestedTimes.call({ isNot: true }, mock, 0)
-        expect(result.pass).toBe(false)
+        expect(result.pass).toBe(true)
```

Comment:
same here .. `expect(mock).not.toBeRequestedTimes(0)` should mean the mock was requested at least 1 time or more to pass but it was not requested so far which should make the assertion `expect(result.pass).toBe(false)` pass.